### PR TITLE
Kill all icon conflicts tbh

### PR DESCRIPTION
**Pull request by Crazylemon64**
I hate icon conflicts

### CODE REVIEW

File: tools/icon_merge_tool/icon_merge_tool.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    if not git_version:
+        print("ERROR: Failed to run git. Make sure it is installed and in your PATH.")
+        return False
+
+    file_conflicts = run_shell_command("git diff --name-only --diff-filter=U").split("\n")
+    icon_conflicts = [path for path in file_conflicts if path[len(path)-3::] == "dmi"]
+
+    for i in range(0, len(icon_conflicts)):
+        print("[{}]: {}".format(i, icon_conflicts[i]))
+    selection = input("Choose maps you want to fix (example: 1,3-5,12):\n")
```
Comment (FlattestGuitar) [inline tools/icon_merge_tool/icon_merge_tool.py]: Icons! >:c

Comment (Crazylemon64) [inline tools/icon_merge_tool/icon_merge_tool.py]: bit**maps** 😄 

ok I'll fix it

**Comment by KasparoVy** [comment]
Sorry for the dumb question-- Whenever I get icon conflicts I always do it like the following: merge from this repo to bring everything up to date, get a new icon file from this repo for the conflicting one and apply the changed icon states from the conflicting icon file to the new file and then drop it in, git add and carry on.

How does this do it?

**Comment by Crazylemon64** [comment]
It uses the dmitool java utility to programatically copy over icon states that changed from both files into "fixed" icon files

**Comment by KasparoVy** [comment]
Excellent, thanks for clearing that up.

**Pull request merged**